sdk/communication/Azure.Communication.Administration/api/Azure.Communication.Administration.netstandard2.0.cs

```diff
@@ -134,6 +134,35 @@ public partial class AcquiredPhoneNumbers
+        public static Azure.Communication.Administration.Models.PhoneNumberCountries PhoneNumberCountries(System.Collections.Generic.IReadOnlyList<Azure.Communication.Administration.Models.PhoneNumberCountry> countries, string nextLink) { throw null; }
+        public static Azure.Communication.Administration.Models.PhoneNumberCountry PhoneNumberCountry(string localizedName, string countryCode) { throw null; }
+        public static Azure.Communication.Administration.Models.PhoneNumberEntities PhoneNumberEntities(System.Collections.Generic.IReadOnlyList<Azure.Communication.Administration.Models.PhoneNumberEntity> entities, string nextLink) { throw null; }
+        public static Azure.Communication.Administration.Models.PhoneNumberEntity PhoneNumberEntity(string id, System.DateTimeOffset? createdAt, string displayName, int? quantity, int? quantityObtained, string status, System.DateTimeOffset? focDate) { throw null; }
+        public static Azure.Communication.Administration.Models.PhoneNumberRelease PhoneNumberRelease(string releaseId, System.DateTimeOffset? createdAt, Azure.Communication.Administration.Models.ReleaseStatus? status, string errorMessage, System.Collections.Generic.IReadOnlyDictionary<string, Azure.Communication.Administration.Models.PhoneNumberReleaseDetails> phoneNumberReleaseStatusDetails) { throw null; }
+        public static Azure.Communication.Administration.Models.PhoneNumberReleaseDetails PhoneNumberReleaseDetails(Azure.Communication.Administration.Models.PhoneNumberReleaseStatus? status, int? errorCode) { throw null; }
+        public static Azure.Communication.Administration.Models.PhoneNumberSearch PhoneNumberSearch(string searchId, string displayName, System.DateTimeOffset? createdAt, string description, System.Collections.Generic.IReadOnlyList<string> phonePlanIds, string areaCode, int? quantity, System.Collections.Generic.IReadOnlyList<Azure.Communication.Administration.Models.LocationOptionsDetails> locationOptions, Azure.Communication.Administration.Models.SearchStatus? status, System.Collections.Generic.IReadOnlyList<string> phoneNumbers, System.DateTimeOffset? reservationExpiryDate, int? errorCode) { throw null; }
+        public static Azure.Communication.Administration.Models.PhonePlan PhonePlan(string phonePlanId, string localizedName, Azure.Communication.Administration.Models.LocationType locationType) { throw null; }
+        public static Azure.Communication.Administration.Models.PhonePlanGroup PhonePlanGroup(string phonePlanGroupId, string localizedName, string localizedDescription) { throw null; }
+        public static Azure.Communication.Administration.Models.PhonePlanGroups PhonePlanGroups(System.Collections.Generic.IReadOnlyList<Azure.Communication.Administration.Models.PhonePlanGroup> phonePlanGroupsValue, string nextLink) { throw null; }
```

Comment:
Some of these types should not be public like `PhonePlanGroups`. Do you mind filing a work item to go through the models and mark the ones that are not needed as `internal`?

Reply:
Added the work item here: https://skype.visualstudio.com/SPOOL/_workitems/edit/2299174